docs/data/pages.ts

```diff
@@ -88,10 +98,7 @@ const pages: MuiPage[] = [
       {
         title: 'Recipes',
         pathname: '/x/react-data-grid/recipes',
-        children: [
-          { pathname: '/x/react-data-grid/recipes-editing', title: 'Editing' },
-          { pathname: '/x/react-data-grid/recipes-row-grouping', title: 'Row grouping' },
-        ],
+        children: [{ pathname: '/x/react-data-grid/recipes-row-grouping', title: 'Row grouping' }],
```

Comment:
What would be the plan with this page/group?
- Rename to `Examples`;
- Refactor in a similar way with a follow-up PR and remove;

Reply:
My plan regarding this page:
- Remove the `Group & Pivot` group and move its pages one level up. This group seems a bit artificial anyway - for example, Aggregation doesn't fit this group. Also, the features in this group are hard to discover (as opposed to other features one level above).
- Split Row grouping into multiple pages and group them, then move the grouping recipes to this group

What do you think?

Reply:
I do understand and agree with the first point. 👌 Really great points regarding `Group & Pivot`.
As far as the second point goes, I feel a bit lost with your provided description. 🤔 
Would the final result be, that the `Recipes` section would be removed and what is currently there would end up under `Row grouping -> Examples`?
Not to mention the other new sections/groups extracted from `Group & Pivot`.
If that's the plan, it does sound very nice to me. 👍 ✅

Reply:
> Would the final result be, that the Recipes section would be removed and what is currently there would end up under Row grouping -> Examples?

Yes. Although we might add a top-level Recipes (or Examples) page for cross-feature examples that we cannot clearly categorize.

For the other pages from the Group & Pivot - we might split some of them too. I think Aggregation might be a good candidate for the page split

Reply:
Thanks for the clarification. 🙏